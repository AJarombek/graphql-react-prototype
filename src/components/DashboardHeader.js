/**
 * Header component for the GitHub dashboard.
 * @author Andrew Jarombek
 * @since 4/10/2020
 */

import React from 'react';

const DashboardHeader = () => {
  return (
    <div className="dashboard-header">
      <p>GraphQL React Prototype</p>
      <p>/</p>
      <p>GitHub</p>
      <p>/</p>
      <p>AJarombek</p>
    </div>
  );
};

export default DashboardHeader;
