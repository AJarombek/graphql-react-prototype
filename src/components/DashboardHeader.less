/**
 * Stylesheet for the DashboardHeader component.
 * Author: Andrew Jarombek
 * Date: 4/10/2020
 */

@import "../vars";

.dashboard-header {
  display: flex;

  font-family: 'Dyslexie', Helvetica, sans-serif;
  font-size: 12px;
  color: @defaultFontColor;

  @media (max-width: 400px) {
    font-size: 9px;
  }

  p {
    margin: 0 4px;
  }

  p:nth-child(5) {
    color: darken(@defaultFontColor, 20%);
  }
}
