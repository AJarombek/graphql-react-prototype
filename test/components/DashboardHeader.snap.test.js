/**
 * Snapshot test for the DashboardHeader component.
 * @author Andrew Jarombek
 * @since 4/25/2020
 */

import React from 'react';
import DashboardHeader from '../../src/components/DashboardHeader';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<DashboardHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
