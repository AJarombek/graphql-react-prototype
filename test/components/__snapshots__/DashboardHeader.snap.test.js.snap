// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="dashboard-header"
>
  <p>
    GraphQL React Prototype
  </p>
  <p>
    /
  </p>
  <p>
    GitHub
  </p>
  <p>
    /
  </p>
  <p>
    AJarombek
  </p>
</div>
`;
